apps/coscrad-frontend-e2e/src/e2e/resources/videos/video.detail.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,287 @@
+import { AggregateType, LanguageCode } from '@coscrad/api-interfaces';
+import { buildDummyAggregateCompositeIdentifier, buildDummyUuid } from '../../../support/utilities';
+
+const videoTitleInLanguage = `Video Title`;
+
+const basicVideoAggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.video,
+    1
+);
+
+const { id: basicVideoId } = basicVideoAggregateCompositeIdentifier;
+
+const mediaItemCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    21
+);
+
+const videoDetailRoute = `/Resources/Videos/${basicVideoId}`;
+
+const validUrl =
+    'https://coscrad.org/wp-content/uploads/2023/05/Rexy-and-The-Egg-_3D-Dinosaur-Animation_-_-3D-Animation-_-Maya-3D.mp4';
+
+describe(`the video detail page`, () => {
+    before(() => {
+        cy.clearDatabase();
+
+        cy.seedTestUuids(50);
+
+        cy.seedDataWithCommand('CREATE_MEDIA_ITEM', {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifier,
+            url: validUrl,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifier,
+        });
+
+        cy.seedDataWithCommand(`CREATE_VIDEO`, {
+            name: videoTitleInLanguage,
+            aggregateCompositeIdentifier: basicVideoAggregateCompositeIdentifier,
+            mediaItemId: mediaItemCompositeIdentifier.id,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: basicVideoAggregateCompositeIdentifier,
+        });
+    });
+
+    beforeEach(() => {
+        cy.visit(videoDetailRoute);
+    });
+
+    describe(`when visiting the detail route with a bogus ID`, () => {
+        beforeEach(() => {
+            cy.visit(`/Resources/Videos/${buildDummyUuid(987)}`);
+
+            cy.getByDataAttribute('not-found');
+        });
+    });
+
+    describe(`when the video is missing optional properties`, () => {
+        it(`should display the video title`, () => {
+            cy.contains(videoTitleInLanguage);
+        });
+    });
+
+    describe.only(`when the video has all properties`, () => {
```

Comment:
Be sure to remove the `.only` before merging down. Also, please ensure Cypress is in a good state before merging, as we have yet to integrate Cypress into the CI flow.